pkg/process/config/yaml_config.go

```diff
@@ -164,6 +164,12 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 		a.EnabledChecks = append(a.EnabledChecks, "TCP queue length")
 	}
 
+	if config.Datadog.GetBool(key(spNS, "enable_process_tracer")) {
+		log.Info("system_probe_config.enable_process_tracer detected, will enable system-probe with process_tracer")
+		a.EnableSystemProbe = true
+		a.EnabledChecks = append(a.EnabledChecks, "Process Tracer")
+	}
+
```

Comment:
Do you forsee options for the process tracer in the future?  If so, we may want to future proof a bit by making the config:

```
system_probe_config:
   process_config: # or process_tracer_config, or something?
        enabled: true
```

Reply:
I don't have concrete idea right now, but I think your suggestion makes sense. We will do a lot more for processes using system-probe in the future that I think it will be more intuitive to have 2 layers.